common/app/views/emailEmbed.scala.html

```diff
@@ -49,7 +49,15 @@
                 });
             }
 
-            revealPrivacyWordingWhenUserInteractsWithEmailField()
+            function informParentOfLoad(loadEvent) {
+                window.parent.postMessage({
+                    subject: 'emailEmbedPageLoaded',
+                    path: window.location.pathname,
+                }, window.origin);
+            }
+
+            window.addEventListener('DOMContentLoaded', informParentOfLoad);
+            revealPrivacyWordingWhenUserInteractsWithEmailField();
```

Comment:
Could we keep the existing line as-is, removing the trailing `;`?

Reply:
ok, will do.